src/plugins/intel_cpu/src/nodes/matmul.cpp

```diff
@@ -527,6 +613,19 @@ MemoryDescPtr MatMul::getSrcMemDesc(dnnl::primitive_desc_iterator &primitive_des
         return DnnlExtensionUtils::makeDescriptor(desc);
 }
 
+MemoryDescPtr MatMul::getDstMemDesc(dnnl::primitive_desc_iterator &primitive_desc_it, size_t idx) {
+    auto desc = primitive_desc_it.dst_desc(idx);
+    return std::make_shared<CpuBlockedMemoryDesc>(
+                        DnnlExtensionUtils::DataTypeToIEPrecision(desc.get_data_type()),
```

Comment:
Data type may be retrieved from the port config of the selected node descriptor. (just faster, since querying dnnl::primitive_desc is a very slow operation).